setup/android_build_env.sh

```diff
@@ -54,5 +63,17 @@ echo "Installing repo"
 sudo curl --create-dirs -L -o /usr/local/bin/repo -O -L https://github.com/akhilnarang/repo/raw/master/repo
 sudo chmod a+x /usr/local/bin/repo
 
-bash ./setup/ccache.sh
-bash ./setup/ninja.sh
+if [[ "$(command -v ccache)" ]]; then
```

Comment:
doesn't need `[[`, can use `[`, as well as an explicit `-z`. Also, how does this handle the case when ccache is not already installed?

Reply:
Yes, I know it doesn't need `[[` but I still chose that for the sake of uniformity in the script, since everywhere else the same has been used. So, either I have to replace `[[` to `[` at other places or use `[[`myself, especially since there is no known performance hits.